Doctrine/Phpcr/BaseSuggestionProvider.php

```diff
@@ -29,22 +30,22 @@
     protected $managerRegistry;
 
     /**
-     * By concatenating the routeBasePath and the url
+     * By concatenating the routeBasePaths and the url
      * we will get the absolute path a route document
      * should be persisted with.
      *
-     * @var string
+     * @var array
      */
-    protected $routeBasePath;
+    protected $routeBasePaths;
 
     /**
      * @param ManagerRegistry $managerRegistry
-     * @param $routeBasePath
+     * @param array           $routeBasePath
      */
-    public function __construct(ManagerRegistry $managerRegistry, $routeBasePath)
+    public function __construct(ManagerRegistry $managerRegistry, array $routeBasePaths)
```

Comment:
is this a BC break? or does php automatically cast a string into an array when needed? i know that `(array) 'bla'` does work but i am unsure about the typehint.